cmd/lncli/invoicesrpc_active.go

```diff
@@ -184,6 +184,14 @@ var addHoldInvoiceCommand = cli.Command{
 				"specified, an expiry of " +
 				"86400 seconds (24 hours) is implied.",
 		},
+		cli.UInt64Flag{
```

Comment:
this doesnt compile. 

Note that you need to add the `invoicesrpc` build tag in order to include this file in your build.

Reply:
Fixed